src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,672 @@
+        "CogPOID":"RECOMMENDED",
+   }
+) }}
+
+
+### Example of * _ephys.json:
+
+<!-- TODO: below there is Procedure.Pharmaceuticals which is not standardized since ATM there is only single pharmaceutical "allowed" and we have no "Procedure" -->
+
+```JSON
+{
+  "PowerLineFrequency": 50,
+  "Manufacturer": "OpenEphys",
+  "ManufacturerModelName": "OpenEphys Starter Kit",
+  "ManufacturerModelVersion": "OEPS-9031",
+  "SamplingFrequency": 30000,
+  "SamplingFrequencyUnit": "Hz",
+  "Location": "Institut de Neurosciences de la Timone, Faculté de Médecine, 27, boulevard Jean Moulin, 13005 Marseille - France",
+  "Software": "Cerebus",
+  "SoftwareVersion": "1.5.1",
+  "Creator": "John Doe",
+  "Maintainer": "John Doe jr.",
+  "Procedure": {
+    "Pharmaceuticals": {
+      "isoflurane": {
+        "PharmaceuticalName": "isoflurane",
+        "PharmaceuticalDoseAmount": 50,
+        "PharmaceuticalDoseUnit": "ug/kg/min"
+      },
+      "ketamine": {
+        "PharmaceuticalName": "ketamine",
+        "PharmaceuticalDoseAmount": 0.1,
+        "PharmaceuticalDoseUnit": "ug/kg/min"
+      }
+    },
+    "Sample": {
+      "BodyPart": "BRAIN",
+      "BodyPartDetails": "Motor Cortex"
+    },
+    "ProbeContours": {
+      "p021": {
+        "Contour": [
+          [0, 0, 0],
+          [0, 10, 0],
+          [1, 11, 0],
+          [2, 10, 0],
+          [2, 0, 0]
+        ],
+        "Unit": "mm"
+      }
+    },
+    "TaskName": "Reach-to-Grasp",
+    "TaskDescription": "A task that involves the reaching of an object and holding it for a specific time"
+  }
+}
+```
+
+## Recording Events (*_events.tsv)
+
+The `*_events.tsv` and corresponding `*_events.json` sidecar files are OPTIONAL and can be used to
+indicate time points of recording events. Each task events file requires a corresponding task data
+file. These events can be internal recording system events, task-related events, or events triggered
+by the experimentalist (for example, manual reward). Note that these events must share a common clock
+with the corresponding ephys recording data. For more details, see the
+Task Events documentation.
+Note that this file can also be used to describe stimulation performed during the recording. For this,
+please follow the iEEG stimulation documentation.
+
+
+## Multi-part recordings
+
+Two different procedures are supported to handle multi-part recordings. In short, the two options are:
+i) each recording is stored in an independent data file, and the corresponding metadata is described
+in the `*_scans.tsv` file; or
+ii) several recordings are stored in a single data file, and the corresponding metadata is described
+in the `*_events.tsv` file.
+These two options are made available to support different usages and habits of the experimenters, as
+well as to benefit from the capability of the supported data formats (NWB and NIX).
+They are described in the following subsections, and made explicit through some of the example data sets.
```

Comment:
My read of the BIDS standard is that (i) _scans.tsv is the more canonical approach. What is the justification for supporting both methods?